hoomd/md/nlist.py

```diff
@@ -369,40 +334,6 @@ def tune_cell_width(self, warmup=200000, min_width=None, max_width=None, jumps=2
 stencil.cur_id = 0
 
 class tree(nlist):
```

Comment:
Sorry I didn't want the docstrings deleted but the documentation removed from the sphinx documentation builds.

Reply:
Added back the deleted docstrings. And removed from the sphinx documentation.